### PR TITLE
ControllerLayer

### PR DESCRIPTION
**Pull request by MihaelaBizgan**

### CODE REVIEW

**Comment by MihaelaBizgan** [comment]
ControllerLayers

File: src/main/java/controller/BaseController.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import org.springframework.web.bind.annotation.RestController;
+
+@RequestMapping(BaseController.API_V1)
+@RestController
+public class BaseController {
+    public static final String API_V1 = "/api/v1";
+
+}
+
+
```
Comment (stefan2212) [inline src/main/java/controller/BaseController.java:13]: Empty line can be removed

File: src/main/java/controller/UserController.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+
+
+
```
Comment (RazvanLoghin) [inline src/main/java/controller/UserController.java:13]: Empty lines can be removed

File: src/main/java/model/User.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+@Data
+@Entity
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private long id;
+
+    @Email(message = "Email should be valid")
```
Comment (stefan2212) [inline src/main/java/model/User.java]: Should be removed

**Review by stefan2212 — CHANGES_REQUESTED** [review]

**Pull request merged**